src/main/java/pl/tomaszdziurko/jvm_bloggers/blog_posts/AggregatedRssFeedProducer.java

```diff
@@ -65,9 +81,10 @@ public SyndFeed getRss(String requestedUrlString) {
             blogPostRepository.findByApprovedTrueOrderByPublishedDateDesc();
```

Comment:
Limiting size of RSS should also result in a smaller collection retrieved from database. Fetching  2000 objects from database for a rss request with `limit`' equal to e.g. `5` is an overkill.